### PR TITLE
LPD-19239 Delete created object definitions at the end of the tests

### PR DESCRIPTION
**Pull request by devpedrotavares**
Related: [LPD-19239](https://liferay.atlassian.net/browse/LPD-19239)

### CODE REVIEW

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -2236,6 +2238,9 @@ public void testDeleteObjectEntry() throws Exception {
 		_objectEntryLocalService.deleteObjectEntry(objectEntry5);
 
 		_assertCount(0);
+
```
Comment (marcelabc) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:2241]: Can you delete `draftObjectDefinition` too?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by devpedrotavares** [comment]
ci:test:sf

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by devpedrotavares** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**